packages/rum-core/src/common/config-service.js

```diff
@@ -89,6 +89,7 @@ class Config {
       transactionSampleRate: 1.0,
       centralConfig: false,
       monitorLongtasks: true,
+      compressPayload: false,
```

Comment:
Thinking about this again, having something like `apiVersion: 2` or similar would be better since we can include all the changes related to v3 before we release the next major.

Reply:
Hmmm, I was thinking in the similar area. apiVersion also sounds good to me. I was thinking in terms of stack version so we can enable breakdown or other configs by default ?

Reply:
We just need to agree on the config name and then we can merge this PR.

Reply:
The stack version is also not a bad idea, although I'm leaning towards api version since we expect it to change less frequently and hopefully we can remove it with version 6.0

Reply:
Sounds good, let's go with apiVersion. I will change it now.